content/docs/installation/orchestrator/orchestrator-helm-repository.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Deploy from repository
```

Comment:
from helm repository, right?

Reply:
yes, was trying to save few characters.

Reply:
at the expense of making thing confusing for dumb people like me x))